app/Config/core.php

```diff
@@ -275,7 +275,7 @@
  * then the value of `Config.timezone` will be used. This feature allows you to set users' timezone just
  * once instead of passing it each time in function calls.
  */
-	//Configure::write('Config.timezone', 'Europe/Paris');
+    //Configure::write('Config.timezone', 'Europe/Paris');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Config.timezone', 'Europe/Paris');" but found "//Configure::write('Config.timezone', 'Europe/Paris');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment